core/client/app/routes/settings/pass-protect.js

```diff
@@ -0,0 +1,44 @@
+import AuthenticatedRoute from 'ghost/routes/authenticated';
+import loadingIndicator from 'ghost/mixins/loading-indicator';
+import CurrentUserSettings from 'ghost/mixins/current-user-settings';
+import styleBody from 'ghost/mixins/style-body';
+
+var SettingsPassProtectRoute = AuthenticatedRoute.extend(styleBody, loadingIndicator, CurrentUserSettings, {
+	classNames: ['settings-view-pass'],
+
+	beforeModel: function () {
+        var feature = this.controllerFor('feature'),
+        self = this;
+
+        if (!feature) {
+            this.generateController('feature');
+            feature = this.controllerFor('feature');
+        }
+
+        return this.currentUser()
+            .then(this.transitionAuthor())
+            .then(this.transitionEditor())
+            .then(function () {
+                return feature.then(function () {
+                    if (!feature.get('passProtectUI')) {
```

Comment:
You're doing a `get` on the promise here, if that's what feature is. You probably wanted to take it as a param in your fulfilled function on the line above